rhea/build/toolflow/xilinx/vivado.py

```diff
@@ -8,6 +8,7 @@
 
 import sys
 import os
+import platform
```

Comment:
What is `platform` I don't see it being used anywhere and I don't see the module/subpackage that defines platform.